src/dirs.proj

```diff
@@ -28,7 +28,12 @@
   </PropertyGroup>
 
   <Import Project="$(ToolsDir)packages.targets" Condition="Exists('$(ToolsDir)packages.targets')" />
-  <Target Name="BuidDirPackages" AfterTargets="Build" DependsOnTargets="BuildPackages" />
+  <PropertyGroup Condition="Exists('$(ToolsDir)packages.targets')">
+    <TraversalBuildDependsOn>
+      $(TraversalBuildDependsOn);
+      BuildPackages;
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
 
```

Comment:
I know this was my change but we should probably consider putting these in the individual targets files now.